.github/workflows/test.yml

```diff
@@ -60,18 +60,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        python: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
What do you think about adding `3.11-dev` while you are at it?

Reply:
Good idea, done!